scripts/mirror.js

```diff
@@ -17,18 +17,28 @@ const compareVersions = require('compare-versions');
 const browsers = require('..').browsers;
 
 const { argv } = require('yargs').command(
-  '$0 <browser> [feature_or_file]',
+  '$0 <browser> [feature_or_path..]',
   'Mirror values onto a specified browser if "version_added" is true/null, based upon its parent or a specified source',
   yargs => {
     yargs
       .positional('browser', {
         describe: 'The destination browser',
         type: 'string',
       })
-      .positional('feature_or_file', {
-        describe: 'The feature, file, or folder to perform mirroring',
-        type: 'string',
-        default: '',
+      .positional('feature_or_path', {
+        describe: 'Features, files, or folders to perform mirroring for',
+        type: 'array',
+        default: [
+          'api',
+          'css',
+          'html',
+          'http',
+          'svg',
+          'javascript',
+          'mathml',
+          'webdriver',
+          'webextensions',
+        ],
       })
       .option('source', {
```

Comment:
Doesn't quite reach within the diff, but would you mind adding a couple of usage examples (with [`.example()`](http://yargs.js.org/docs/#api-reference-examplecmd-desc)) toward the end of the yargs stuff? Two or three would be great (e.g., with a feature, with a path, with more than one of something).